cmd/webhook.go

```diff
@@ -218,9 +223,12 @@ func createPatch(pod *corev1.Pod, sidecarConfigTemplate *Config, annotations map
 		panic(err.Error())
 	}
 
-	// Retrieves the list of secrets
-	secretList, _ := clientset.CoreV1().Secrets(pod.Namespace).List(context.Background(), metav1.ListOptions{})
-
+	// Retrieves the configmap containing the list of shares. Must loop through this
+	// Format is "filer1": '["share1", "share2"]'
+	svmShareList, errorSvm := clientset.CoreV1().ConfigMaps(pod.Namespace).Get(context.Background(), requestConfigMapName, metav1.GetOptions{})
+	if k8serrors.IsNotFound(errorSvm) {
+		return []byte(""), errorSvm
```

Comment:
I think by convention, you should return the zero value of the type you are returning when dealing with errors. So i think this should return "nil, error", since an array isnt a primitive type. Unless you need to return a byte array for "".